config/go-app-ussd_tb_check.afr_ZA.json

```diff
@@ -130,7 +130,7 @@
     ],
     "FAQ for more info on TBCheck and the research.": [
         null,
-        "FAQ, vir meer oor TBCheck asook die navorsing daaroor."
+        "FAQ vir meer information."
```

Comment:
I think this should be "informasie"?

Reply:
@erikh360 

I copied from their feedback and pasted as is